accountsplus/templates/accounts/locked_out.html

```diff
@@ -0,0 +1,31 @@
+{% load bootstrap3 %}
+
+{% block content %}
+
+    <div class="col-lg-6 col-lg-offset-3 col-md-6 col-md-offset-3 locked-out-container">
+        <div class="login-message">
+            Your login is blocked due to multiple failing login attempts. Solve the following captcha to get unblocked!
```

Comment:
well, both do the same job so you can you what it fits you more, check this
http://stackoverflow.com/questions/17621770/django-i18n-blocktrans-vs-trans